projects/plugins/jetpack/_inc/lib/functions.wp-notify.php

```diff
@@ -1,427 +0,0 @@
-<?php
```

Comment:
It may be best to keep the file around for one more release, to avoid fatals on sites with an aggressive caching. I think we can keep the file and only mark it as deprecated. See `projects/plugins/jetpack/modules/cloudflare-analytics/cloudflare-analytics.php` for an example of what I mean.

Reply:
Makes sense!

Reply:
Addressed in 0711b660f398b04835384ea662a9bf2a406c5475